src/components/swap/SwapBox.vue

```diff
@@ -208,6 +214,13 @@ export default defineComponent({
     toggleshowMaxTooltip() {
       this.showMaxTooltip = !this.showMaxTooltip;
     },
+    selectAddressType(type) {
```

Comment:
I'm sorry I'm a bit confused, but I don't see where do you use the selected address in the swapData. Am I missing something?

Reply:
This method is used to change the type of address, connected -> Metamask or different -> Any other address added by user.

Reply:
and where do you send that in the call to the api?